File: Curso Nélio-Alves/Código em C/exercicio2-2.c
#include <stdio.h>
#include <math.h>
int main(){
    float a, b, c, delta, x1, x2;
    scanf("%f", &a);
    scanf("%f", &b);
    scanf("%f", &c);
    if(a == 0){
        printf("Impossivel calcular");
    }else{
        delta = pow(b, 2) - 4 * a * c;
        if(delta < 0){
            printf("Impossivel calcular");
        }else{
            x1 = (-b + sqrt(delta)) / (2 * a);
            printf("X1 = %.5f\n", x1);
            x2 = (-b - sqrt(delta)) / (2 * a);
            printf("X2 = %.5f\n", x2);
        }
    }
    return 0;
}

File: Curso Nélio-Alves/Código em Python/exercicio2-2.py
from cmath import sqrt
# Pegar informações da equação
val = input()

# Tratar valores pegos
vet = val.split()

# Coletando valor A
a = float(vet[0])

# Coletando valor B
b = float(vet[1])

# Coletando valor C
c = float(vet[2])

if a==0:
    print("Impossivel calcular")
else:
    delta = pow(b, 2) - (4*a*c)
    if delta < 0:
        print("Impossivel calcular")
    else:
        r1 = ((-b) + sqrt(delta))/(2*a)
        print("X1 = {:.5f}".format(r1.real))
        r2 = ((-b) - sqrt(delta))/(2*a)
        print("X2 = {:.5f}".format(r2.real))
